resources/assets/less/variables.less

```diff
@@ -255,8 +255,8 @@
 @diff-normal: @blue;
 @diff-hard: @yellow;
 @diff-insane: @pink;
-@diff-expert: @purple;
-@diff-expert-plus: #000;
+@diff-expert: @purple-light;
```

Comment:
not for everything, just the light variation...

Reply:
you saw nothing.